app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedViewModel.kt

```diff
@@ -1,34 +1,70 @@
 package com.lightricks.feedexercise.ui.feed
 
+import android.util.Log
 import androidx.lifecycle.*
 import com.lightricks.feedexercise.data.FeedItem
 import com.lightricks.feedexercise.data.FeedRepository
 import com.lightricks.feedexercise.network.FeedApiService
 import com.lightricks.feedexercise.util.Event
 import java.lang.IllegalArgumentException
+import com.lightricks.feedexercise.R
 
 /**
  * This view model manages the data for [FeedFragment].
  */
 open class FeedViewModel(private val feedRepository: FeedRepository) : ViewModel() {
-    private val isLoading = MutableLiveData<Boolean>()
-    private val isEmpty = MutableLiveData<Boolean>()
-    private val feedItems = MediatorLiveData<List<FeedItem>>()
+    private val TAG = "FeedViewModel"
```

Comment:
Add the const keyword and move it to the companion object. Since this field can be static in the class.